common/types/src/main/java/io/helidon/common/types/TypeInfo.java

```diff
@@ -16,193 +16,889 @@
 
 package io.helidon.common.types;
 
-import java.util.List;
-import java.util.Map;
+import java.util.Objects;
 import java.util.Optional;
-import java.util.Set;
+
+import io.helidon.common.Errors;
 
 /**
  * Represents the model object for a type.
+ *
+ * @see #builder()
  */
-public interface TypeInfo {
-    /**
-     * The {@code public} modifier.
-     */
-    String MODIFIER_PUBLIC = "PUBLIC";
-    /**
-     * The {@code protected} modifier.
-     */
-    String MODIFIER_PROTECTED = "PROTECTED";
-    /**
-     * The {@code private} modifier.
-     */
-    String MODIFIER_PRIVATE = "PRIVATE";
+@io.helidon.common.Generated(value = "io.helidon.builder.processor.BlueprintProcessor",
```

Comment:
remove generated

Reply:
fixed (all types in common/types I hope)